mk/docker.mk

```diff
@@ -11,13 +11,15 @@ define build_image
 $(DOCKER_REGISTRY)/$(1):$(BUILD_INFO_VERSION)$(if $(IMAGE_ARCH_TAG_ENABLED),-${GOARCH},)
 endef
 
+export KUMA_BASE_IMAGE ?= kumahq/base-debian11:1.0.0
+export KUMA_STATIC_IMAGE ?= kumahq/static-debian11:1.0.0
```

Comment:
Do we really want to use semver here when in practice this tag always move?

Reply:
Why always? I think it will really rarely move. What would you suggest using?

Reply:
latest. You always try to rebuild the image it's never pushed or antyhign

Reply:
What I meant is from the way this is used you'll be able to change the dockerfile and this will not change this version number and semVer brings a false sense of being static. Maybe even have a tag: `locally-built` would make this clearer and add a comment to indicate this is built inline

Reply:
> Maybe even have a tag: locally-built or no-push would make this clearer and add a comment to indicate this is built inline.

I'm the most convinced by this.